Aquality.Selenium.Template/Aquality.Selenium.Template.NUnit/Steps/ContactUsPageSteps.cs

```diff
@@ -0,0 +1,106 @@
+﻿using NUnit.Framework;
+using System;
+using Aquality.Selenium.Browsers;
+using Aquality.Selenium.Template.Constants;
+using Aquality.Selenium.Template.Enums;
+using Aquality.Selenium.Template.Forms.Pages;
+using Aquality.Selenium.Template.Utilities;
+using Aquality.Selenium.Template.NUnit.Extensions;
+using Aquality.Selenium.Template.Models;
+using Aquality.Selenium.Template.Extensions;
+using Aquality.Selenium.Template.NUnit.Constants;
+
+namespace Aquality.Selenium.Template.NUnit.Steps
+{
+    public class ContactUsPageSteps : BaseSteps
+    {
+        private readonly ContactUsPage contactUsPage;
+        private readonly ContactUsInfo contactUsInfo;
+        const string TitleLabelText = "We are glad to hear from you!";
```

Comment:
It's better to move such text into constants class e.g. TitleConstants

Reply:
Done